modules/react/skeleton/lib/skeleton.tsx

```diff
@@ -4,11 +4,11 @@ import styled from '@emotion/styled';
 
 import {accessibleHide, createComponent, StyledType} from '@workday/canvas-kit-react/common';
 
-import SkeletonHeader from './parts/skeletonHeader';
-import SkeletonShape from './parts/skeletonShape';
-import SkeletonText from './parts/skeletonText';
+import {SkeletonHeader} from './parts/skeletonHeader';
+import {SkeletonShape} from './parts/skeletonShape';
+import {SkeletonText} from './parts/skeletonText';
 
-interface SkeletonProps {
+export interface SkeletonProps {
```

Comment:
Also, I had to export `SkeletonProps` in order for the `ArgsTable` to show up for `Skeleton`.